install/kubernetes/helm/istio/charts/galley/values.yaml

```diff
@@ -4,6 +4,7 @@
 enabled: true
 replicaCount: 1
 image: galley
+sidecar: true
```

Comment:
will this still work if sidecar is false?

Reply:
No, given that this is a WIP PR. The port # changes, along with the mTLS settings. Both need to be accounted for, which this PR doesn't do yet.

Reply:
sgtm, was just pointing it out in case it helped, I saw it was a WIP though

Reply:
Sorry, didn't mean to be curt. It took me a while to get the whole templating thing just right to get it ported from the installer and merging with the istio API idioms. This is just to get the hello world up and running. I haven't done the full gamut of "what can go wrong here" walkthrough yet. I am sure more things will come up.